data/templates/default.latex

```diff
@@ -1,4 +1,4 @@
-\PassOptionsToPackage{unicode=true,$for(hyperrefoptions)$$hyperrefoptions$$sep$,$endfor$}{hyperref} % options for packages loaded elsewhere
+\PassOptionsToPackage{unicode=true$if(hyperrefoptions)$,$for(hyperrefoptions)$$hyperrefoptions$$sep$,$endfor$$endif$}{hyperref} % options for packages loaded elsewhere
```

Comment:
Why not
```
unicode=true$for(hyperrefoptions)$,$hyperrefoptions$$endfor$
```

Reply:
This is much cleaner and I wasn't aware of. Thx.